src/boot/bootctl-util.h

```diff
@@ -8,3 +8,11 @@ const char *get_efi_arch(void);
 int get_file_version(int fd, char **ret);
 
 int settle_entry_token(void);
+
+static inline const char* get_conf_root(void)
```

Comment:
hmm, this named a bit generic and hence cryptic. Maybe just rename this `etc_kernel()`? I think that makes it very clear what this is...